hewr/R/to_hub_quantile_table.R

```diff
@@ -55,7 +55,7 @@ to_hub_quantile_table <- function(model_batch_dir) {
     quantiles_paths <- fs::dir_ls(
       model_run_dir,
       recurse = TRUE,
-      glob = "*_quantiles.parquet"
+      glob = "*_quantiles_e.parquet"
```

Comment:
I think we want this in case we add other signal quantile forecasts in the future 
```suggestion
      glob = "*_quantiles_*.parquet"
```